proparse/src/main/java/org/prorefactor/treeparser/ParseUnit.java

```diff
@@ -28,6 +31,7 @@
 import org.prorefactor.macrolevel.PreprocessorEventListener.EditableCodeSection;
 import org.prorefactor.proparse.IntegerIndex;
 import org.prorefactor.proparse.ProParser;
+import org.prorefactor.proparse.ProParserTokenTypes;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.prorefactor.proparse.ProParserTokenTypes'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AUselessImportCheck)